content/lessons/outro/index.tsx

```diff
@@ -0,0 +1,23 @@
+'use client'
+import { useTranslations } from 'hooks'
+import { TextImage } from 'ui'
+
+export default function Outro({ lang }) {
```

Comment:
Should we place Outro in the `/done` folder?

Reply:
The `done` page symbolizes the end of the demo, whereas the `outro` page symbolizes the end of Chapter-1.
So they serve different purposes, and IMO is better to keep them in separate folders.

However, if you think it is better to keep them together, I would love to listen to your take on this.

Reply:
Ok! That makes sense to me!